Source/JavaScriptCore/wasm/WasmCallingConvention.h

```diff
@@ -162,6 +162,7 @@ class WasmCallingConvention {
             case TypeKind::Funcref:
             case TypeKind::RefNull:
             case TypeKind::Ref:
+            case TypeKind::Arrayref:
```

Comment:
I think that this isn't quite the right change here, because `arrayref` as a heap type should typically not appear as an actual type kind (it should be a `RefNull`/`Ref`). The reason it's needed is I think due to a use of `Types::Arrayref` below which can be changed.